pkgs/top-level/ocaml-packages.nix

```diff
@@ -102,24 +106,32 @@ let
 
     brr = callPackage ../development/ocaml-modules/brr { };
 
+    buildDunePackage = callPackage ../build-support/ocaml/dune.nix {}; # Libs
+
+    buildOasisPackage = callPackage ../build-support/ocaml/oasis.nix { }; # Libs
+
```

Comment:
Those are better kept out of order (IMHO).

Reply:
Can I add it to end? People won't change it. Less scrolling for adding new packages.

Reply:
Done